aws/biganimal-csp-setup

```diff
@@ -0,0 +1,319 @@
+EOF
+  )
+  echo "$biganimal_policy" > "${TMPDIR}"/biganimal_policy.json
+
+  # create IAM policy using aws-cli
+  echo -e "Creating AWS IAM policy ${GREEN}biganimal-${suffix_policy_name}${NC} within AWS account ${GREEN}${account}${NC}..."
+  aws iam create-policy --policy-name "biganimal-$suffix_policy_name" --policy-document file://"${TMPDIR}"/biganimal_policy.json \
```

Comment:
suggest to pass `--output json` to `aws iam create-policy`, this way can make the script run without relying on the user's local configuration 

`aws iam create-role` is the same as

Reply:
agreed, I did the suggested changes. Thanks!